docs/docs-content/profiles/cluster-profiles/create-cluster-profiles/define-profile-variables.md

```diff
@@ -0,0 +1,153 @@
+it is subject to change.
+
+:::
+
+When defining a cluster profile variable, you can set specific constraints on the expected values, such as format,
+optionality, masking, and so on, to ensure scalable, error-free cluster deployments.
+
+:::info
+
+You can use profile variables with any number of packs, manifests, and Helm charts, but only in the scope of their
+parent cluster profile. If you want to create placeholders to use across different cluster profiles, consider using
+[Palette Macros](/clusters/cluster-management/macros/).
+
+:::
+
+This guide explains how you can define and manage cluster profile variables.
+
+## Prerequisites
+
+- The `clusterProfile.create` and `clusterProfile.update` permissions to create and update cluster profiles. Refer to
+  [Roles and Permissions](/user-management/palette-rbac/project-scope-roles-permissions#cluster-profile-admin) for more
+  information.
+
+- An in-progress or already created cluster profile. The cluster profile must either have the **Edge Native**
+  infrastructure or be an edge-based add-on profile.
+
+## Define Profile Variables
+
+:::warning
+
+Palette does not support nesting profile variables within macros or other profile variables. Also, you cannot define
+profile variables for the `pack.content` and `system.uri` parameters because the [Palette CLI](/palette-cli/) populates
+them automatically.
+
+:::
+
+You can define profile variables both while creating and for the already created cluster profiles. To define profile
+variables [while creating a cluster profile](/profiles/cluster-profiles/create-cluster-profiles/), start following this
+guide from step three at the **Profile Layers** stage of cluster profile creation.
+
+1.  Log in to Palette.
+
+2.  Navigate to the cluster profile for which you want to define profile variables.
+
+3.  In the upper-right corner, click **Variables** and, on the **Profile variables** pane, click **Create variable**.
+
+    ![Palette with the Variables button highlighted.](/profiles_create-cluster-profiles_define-profile-variables_open-profile-variables.png)
+
+4.  Enter the variable name in the `{{.spectro.var.<variable_name>}}` format. You will use this name when adding
+    variables to layer configurations. The name must be unique within the parent cluster.
+
+5.  Enter a variable display name that Palette will display during cluster deployment. The display name must be unique
+    within the parent cluster.
+
+6.  Optionally, enter the variable description.
+
+7.  Select a data format that the variable will expect. The following table describes the available data formats.
+
+    | **Format** | **Description**                                                                                        |
+    | ---------- | ------------------------------------------------------------------------------------------------------ |
+    | String     | Custom string of text.                                                                                 |
```

Comment:
```suggestion
    | String     | Custom text input.                                                                                 |
```
Maybe this? Using the term in the definition is something we should avoid 😅

Reply:
You're right, fixed it and committed separately ✨